### PR TITLE
Update ember-data.md

### PR DESCRIPTION
**Pull request by snaquaye**

### CODE REVIEW

**Comment by snaquaye** [comment]
Spent hours trying to make this work. This should help people like me. :)

File: source/tutorial/ember-data.md [hunk 1]
```diff
@@ -52,12 +52,27 @@ Ember Data can be configured to save data in a variety of ways, but often it is
...
 This will allow us to create fake data to work with while developing our app and mimic a running backend server.
 
+
 Let's start by installing Mirage:
 
 ```shell
 ember install ember-cli-mirage
 ```
 
+NOTE: To use Mirage, you would have to enable it. See snipet below
```
Comment (locks) [inline source/tutorial/ember-data.md]: You mispelled `snippet`, see failing test suite.

**Comment by rtablada** [comment]
Ember CLI Mirage is enabled by default on `development` and `testing` environments so this shouldn't be needed.

File: source/tutorial/ember-data.md [hunk 2]
```diff
@@ -52,12 +52,27 @@ Ember Data can be configured to save data in a variety of ways, but often it is
 For this tutorial, we will use [Mirage](http://www.ember-cli-mirage.com).
 This will allow us to create fake data to work with while developing our app and mimic a running backend server.
 
+
```
Comment (rtablada) [inline source/tutorial/ember-data.md]: Should't need this extra line

**Comment by michaelrkn** [comment]
My experience is consistent with @rtablada's comment - I didn't need to explicitly enable Mirage when I went through the tutorial. I'm going to close this out, but @snaquaye, if you can provide an example repo demonstrating the problem, I'd be happy to look into it more. Thanks!

**Pull request closed without merging**